pkg/apis/db/v1alpha1/cassandrabackup_types.go

```diff
@@ -47,21 +48,24 @@ type CassandraBackupSpec struct {
 	Secret                string `json:"secret,omitempty"`
 }
 
-type BackupState string
+type BackupConditionType string
 
 const (
-	BackupRunning   BackupState = "RUNNING"
-	BackupCompleted BackupState = "COMPLETED"
-	BackupFailed    BackupState = "FAILED"
+	BackupRunning   BackupConditionType = "RUNNING"
```

Comment:
@cscetbon all possible states an operation can be in: https://github.com/instaclustr/instaclustr-commons/blob/master/src/main/java/com/instaclustr/operations/Operation.java#L175

Reply:
Yes @smiklosovic  but we're not using the other types in our tests. In our use, `CANCELLED` should never happen, you confirm ?

Reply:
yeah it shouldnt occur in your case.